src/ImageProcessor/Colors/ColorSpacialTransforms.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="ColorSpacialTransforms.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageProcessor
+{
+    public struct ColorSpacialTransforms
+    {
+        public static Color Multiply(Color source, Color destination)
+        {
+            if (destination == Color.Black)
+            {
+                return Color.Black;
+            }
+            if (destination == Color.White)
+            {
+                return source;
+            }
+            return new Color(
+                (source.R * destination.R).Clamp(0f, 1f),
```

Comment:
Once you use the correct partial you can then multiply the backing vectors. Don't limit as I want to support HDR